src/main/java/de/unistuttgart/crosswordbackend/service/GameResultService.java

```diff
@@ -0,0 +1,55 @@
+package de.unistuttgart.crosswordbackend.service;
+
+import de.unistuttgart.crosswordbackend.clients.ResultClient;
+import de.unistuttgart.crosswordbackend.data.GameResultDTO;
+import de.unistuttgart.crosswordbackend.data.OverworldResultDTO;
+import feign.FeignException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+/**
+ * This service handles the logic for the GameResultController.class
+ */
+@Service
+@Slf4j
+public class GameResultService {
+
+  @Autowired
+  ResultClient resultClient;
+
+  /**
+   * Creates a OverworldResultDTO and sends it to the overworld backend.
+   *
+   * @param gameResult extern gameResultDTO
+   * @param userId Id of the user
+   * @param accessToken accessToken of the user
+   * @throws IllegalArgumentException if at least one of the arguments is null
+   */
+  public void submitGameResult(GameResultDTO gameResult, String userId, String accessToken) {
+    if (gameResult == null || userId == null || accessToken == null) {
+      throw new IllegalArgumentException("gameResultDTO or userId is null");
+    }
+    int score = 100 * gameResult.getCorrectTiles() / gameResult.getNumberOfTiles();
```

Comment:
If I see this correctly, the user has full control of the API call how many correct tiles, and how many tiles he can specify. Check preconditions, like correct tiles < number of tiles